src/containerapp/azext_containerapp/_params.py

```diff
@@ -137,9 +138,22 @@ def load_arguments(self, _):
         c.argument('service_type', help="The service information for dev services.")
         c.ignore('service_type')
 
+    with self.argument_context('containerapp create', arg_group='GitHub Repository') as c:
+        c.argument('repo', help='Create an app via GitHub Actions in the format: https://github.com/<owner>/<repository-name> or <owner>/<repository-name>')
```

Comment:
Update to `The repo used to create app via Github Actions. Typically in the format of https://github.com/<owner>/<repository-name> or <owner>/<repository-name> `?

Reply:
The format is `https://github.com/<owner>/<repository-name>`